### DISCUSSION TITLE
Queue system not working as intended

### DISCUSSION THREAD

**Discussion by VanLyden** [c0nD/FluffyBOT#62] [Bug Report]
I know you guys updated the queue system to accommodate for when a sweeper is needed, but now it is not pinging the next person when a boss is swept. I've attached an example below. Best of luck fixing this! <3

![image](https://user-images.githubusercontent.com/115385250/224476702-d92df1e1-0c4b-4fa4-a415-a7b85f2c0a5a.png)
![image](https://user-images.githubusercontent.com/115385250/224476714-81787c54-b5c9-4a29-a305-f966870c4ac7.png)

**Comment by c0nD** [comment]
Thanks for the report -- we'll do our best to investigate this issue and fix it ASAP.

**Comment by c0nD** [comment]
Should be fixed with PR #66 

(Keeping thread open incase it pops back up.)
